config/v1/types_authentication.go

```diff
@@ -59,12 +59,10 @@ type AuthenticationSpec struct {
 	// serviceAccountIssuer is the identifier of the bound service account token
 	// issuer.
 	// The default is https://kubernetes.default.svc
-	// WARNING: Updating this field will result in the invalidation of
-	// all bound tokens with the previous issuer value. Unless the
-	// holder of a bound token has explicit support for a change in
-	// issuer, they will not request a new bound token until pod
-	// restart or until their existing token exceeds 80% of its
-	// duration.
+	// WARNING: Updating this field will not result in immediate invalidation of all bound tokens with the
+	// previous issuer value. Instead, the tokens issued by previous service account issuer will continue to
+	// be trusted for a time period chosen by the platform (currently set to 24h).
```

Comment:
You should explicitly include the sentence `This time period is subject to change over time`, we use it commonly where we expect values to change over time, good to use the common verbiage

Reply:
sentence added